core/state/statedb.go

```diff
@@ -349,11 +364,16 @@ func (s *StateDB) Reset(root common.Hash) error {
 	s.thash = common.Hash{}
 	s.bhash = common.Hash{}
 	s.txIndex = 0
-	s.logs = make(map[common.Hash][]*types.Log)
 	s.logSize = 0
 	s.preimages = make(map[common.Hash][]byte)
 	s.clearJournalAndRefund()
 
+	// Clear all entries
```

Comment:
would it be faster to initialize the map again? like how it was done previously

Reply:
https://stackoverflow.com/questions/49355345/how-to-clean-a-sync-map

Can be a race condition on the `map` object

Reply:
an operation might be reading from the same map while the initialization is taking place